src/sentry/models/groupassignee.py

```diff
@@ -134,12 +134,12 @@ def assign(
         self,
         group: Group,
         assigned_to: Team | RpcUser | User,
-        acting_user: User | None = None,
+        acting_user: RpcUser | User | None = None,
```

Comment:
Down below we call `create_group_activity` which expects `RpcUser`
https://github.com/getsentry/sentry/blob/0d337eb5e24a7c70a2d8ea9c896b4228ee71e5ce/src/sentry/models/activity.py#L81-L85